integration/test/Test/Federation.hs

```diff
@@ -68,12 +69,13 @@ testNotificationsForOfflineBackends = do
     bindResponse (postConversation delUser (defProteus {qualifiedUsers = [otherUser, downUser1]})) $ \resp ->
       resp.status `shouldMatchInt` 533
 
-    -- Adding users to an up backend conversation should work even when one of
-    -- the participating backends is down
-    otherUser2 <- randomUser OtherDomain def
-    connectUsers delUser otherUser2
-    bindResponse (addMembers delUser upBackendConv [otherUser2]) $ \resp ->
-      resp.status `shouldMatchInt` 200
+    -- Adding users to an up backend conversation should work not work when one
```

Comment:
"should work not work" - must be a left-over in here, but you tell which is which.

Reply:
Good catch, It is "should not work".